extensions/wikia/CuratedContent/CuratedContentController.class.php

```diff
@@ -358,7 +359,7 @@ function ( $ret, $item ) {
 					$ret[] = [
 						'title' => $item['title'],
 						'image_id' => $imageId,
-						'image_url' => CuratedContentSpecialController::findImageIfNotSet( $imageId )[1]
+						'image_url' => CuratedContentHelper::findImage( $imageId )[1]
```

Comment:
maybe it's better to return associative array? It would give some idea what are you returning from this function.
Also I'm thinking if it's not worth to create two functions, find image by id (which would return URL) and find image by article id and it would return imageId,Url array?